server-kotlin/src/main/kotlin/digital/cesko/Application.kt

```diff
@@ -48,7 +48,7 @@ class Application {
     fun corsConfigurer(): WebMvcConfigurer {
         return object : WebMvcConfigurer {
             override fun addCorsMappings(registry: CorsRegistry) {
-                registry.addMapping("/**").allowedOrigins("*")
+                registry.addMapping("/**").allowedOrigins("http://localhost:4200").allowCredentials(true)
```

Comment:
Je nějaký důvod pro tuhle změnu? Přísný CORS nepotřebujeme a už si nepamatuji všechny scénáře vývoje.

Reply:
Pokud si dobre pamatuji, tak Chrome vyhazoval tuto hlasku: `Cannot use wildcard in Access-Control-Allow-Origin when credentials flag is true.`, takze jsem tam musel explicitne nastavit ten origin pro vyvoj. Tykalo se to prihlasovani do administrace.